src/geometries/WireframeGeometry.js

```diff
@@ -124,7 +124,8 @@ function isUniqueEdge( start, end, edges ) {
 
 	} else {
 
-		edges.add( hash1, hash2 );
+		edges.add( hash1 );
+		edges.add( hash2 );
```

Comment:
This is actually an import fix since `hash2` was never added to `edges`.

Reply:
Wow...

Reply:
So it means that we were *still* generating duplicated lines?

Reply:
Both hashes should be part of `edges`. The new code ensures that this is the case.

Reply:
>Both hashes should be part of edges.

I do not think that is necessary.

Reply:
After a second look it's indeed sufficient to only store a single edge. The coincident edge is not required in the set since when an edge is tested, `isUniqueEdge()` generates two hashes for testing the actual and the coincident edge.

However, I don't think the logic is "wrong". Storing both edges allows a potential early out in `isUniqueEdge()` ( you can safe a call of `Set.has()`).

Reply:
>After a second look it's indeed sufficient to only store a single edge. 

As I said, the coincident edge did not have been added to the set in this PR.

If you insist on adding both edges to the set, then you can compute `hash2` only when it is added. Your choice. :-)